IdentityCore/src/MSIDError.m

```diff
@@ -36,6 +36,7 @@
 NSString *MSIDTokenProtectionRequired = @"MSIDTokenProtectionRequired";
 NSString *MSIDBrokerVersionKey = @"MSIDBrokerVersionKey";
 NSString *MSIDServerUnavailableStatusKey = @"MSIDServerUnavailableStatusKey";
+NSString *MSIDThrottlingCacheHitKey = @"MSIDThrottlingCacheHitKey";
```

Comment:
what does this key indicate, sorry I missed the context ?

Reply:
If an error is returned from MSAL's throttling cache instead of server response, it will have this extra key in its userinfo.

Reply:
I see , do we have any example of such error scenarios ?

Reply:
It is a common scenario:
1st silent request goes to eSTS as usual, get back error in throttling conditions (429, invalid_grant,etc...)
client repeat the same request 2nd time in the throttling windows, it will get back error from throttling cache immediately without trying server.
The purpose of this change is providing extra information for client to be aware of the error source. For example, they shouldn't report its to their telemetry (or still reporting but should put into a new category). Or once they know they are in throttled state, should priority the next request for most important resource.

Reply:
Got it, thanks Hieu. When do we get error from throttling cache without trying with server ?

Reply:
Do you mean other than the example I gave above? I don't get the question yet.